hack/make.ps1

```diff
@@ -126,6 +126,25 @@ Function Check-InContainer() {
         Write-Host ""
         Write-Warning "Not running in a container. The result might be an incorrect build."
         Write-Host ""
+        return $False
+    }
+    return $True
+}
+
+# Utility function to warn if the version of go is correct. Used for local builds
+# outside of a container where it may be out of date with master.
+Function Verify-GoVersion() {
+    Try {
+        $goVersionDockerfile=(Get-Content ".\Dockerfile" | Select-String "ENV GO_VERSION").ToString().Split(" ")[2]
```

Comment:
To add, if the versions of go have to be different between Windows and Linux (and I sincerely hope that isn't the case) in the future, we will see other CI errors thrown up, so I don't think checking dockerfile.Windows makes sense at this point.